packages/KSelect/KSelect.vue

```diff
@@ -292,6 +309,13 @@ export default {
     loading: {
       type: Boolean,
       default: false
+    },
+    /**
+     * A flag for clearing selection when appearance is 'select'
+     */
+    clearable: {
+      type: Boolean,
+      default: true
```

Comment:
This should be `false` by default, so it doesn't change the current implementation